### ISSUE TITLE
Crash telegram desktop

### ISSUE THREAD

**Issue by VRT-secure** [aunetx/blur-my-shell#462]
Hello comrades,  the extension  crash my "telegram desktop" when I work with touchpad on my laptop. 

My OS is Fedora 38 with Wayland.

**Comment by CorvetteCole** [comment]
I don't think this extension could cause that, given that Telegram desktop is Qt based and doesn't utilize any blurring functions. Do you have any logs that seem to indicate Blur My Shell is the problem?
